### PR TITLE
Enable benchmark_app tests

### PR DESCRIPTION
**Pull request by Wovchena**

### CODE REVIEW

**Comment by ilya-lavrenov** [comment]
BTW, can we move private sample tests to main repo?
looks like there we have more tests than publicly

**Comment by Wovchena** [comment]
I'm going to merge private tests in the next PR

File: tests/samples_tests/smoke_tests/README.md [hunk 1]
```diff
@@ -19,7 +19,7 @@ cmake -DCOMPONENT=tests -DCMAKE_INSTALL_PREFIX=<INSTALL_DIR> -P cmake_install.cm
 a. Required:
     - IE_APP_PATH : coomon path to C++ and C samples, e.g. '<INSTALL_DIR>/samples_bin'
     - IE_APP_PYTHON_PATH : path to python IE samples, e.g. '<INSTALL_DIR>/samples/python/'
-    - IE_APP_PYTHON_TOOL_PATH : path to python IE tools for benchmark_app, e.g. '<INSTALL_DIR>/tools/'
```
Comment (ilya-lavrenov) [inline tests/samples_tests/smoke_tests/README.md:22]: why this way does work anymore?

Comment (Wovchena) [reply tests/samples_tests/smoke_tests/README.md:22]: This was the only mention of `IE_APP_PYTHON_TOOL_PATH` in the whole repo

**Review by ilya-lavrenov — APPROVED** [review]

**Pull request merged**